src/base/format/tree.cc

```diff
@@ -194,8 +194,11 @@ inline std::unique_ptr<AbstractNode> Tree::parse_type_spec(const char*& format_s
 	case 'b':
 		return std::unique_ptr<AbstractNode>(new BooleanNode(flags, min_width, precision));
 	case 'f':
-		return std::unique_ptr<AbstractNode>(new FloatNode(
-		   flags, min_width, precision == kInfinitePrecision ? kDefaultFloatPrecision : precision));
+		bool dynamic_precision;
+		dynamic_precision = (precision == kInfinitePrecision);
```

Comment:
```suggestion
		const bool dynamic_precision = (precision == kInfinitePrecision);
```

Reply:
Just add `{}` around this `case`. This should always be done when declaring local variables in a switch statement.
```c++
switch (...) {
case 'c':
	...
	break;
case 'P': {
	bool localvar = ...;
	...
} break;
}
```

Reply:
Thanks!